server/test/utils.ts

```diff
@@ -0,0 +1,94 @@
+import { rand, randEmail, randNumber, randUserName, randQuote, randBrand } from '@ngneat/falso';
+
+import { User } from 'server/models';
+
+// user interface
+interface UserObject {
+  email?: string;
+  discord_user_id?: string;
+  discord_name?: string;
+  bio?: string;
+  twitter_username?: string;
+  linkedin_url?: string;
+  github_username?: string;
+  website?: string;
+}
+
+// https://stackoverflow.com/questions/39494689/is-it-possible-to-restrict-number-to-a-certain-range/39495173#39495173
+// Tim's fault for telling me this is possible
+type Enumerate<N extends number, Acc extends number[] = []> = Acc['length'] extends N
+  ? Acc[number]
+  : Enumerate<N, [...Acc, Acc['length']]>;
+
+type IntRange<F extends number, T extends number> = Exclude<Enumerate<T>, Enumerate<F>>;
```

Comment:
wheee 🙃 i think these are probably best in a separate types file, since it's pretty general-purpose and may be used for other things

Reply:
maybe `server/types/utils.ts`?